### PR TITLE
Calibrate line spacing values.

### PR DESCRIPTION
**Pull request by hedgerwang**
I just realized that all the line spacing values exported by Google can't be used directly. 

After doing a precise measurement of different line spacing values exported from Google, I update our own logic that normalized the line spacing values. 

Also, I change the default line spacing value from "125" to "138%", so it uses the same line spacing value as Google dos.

### CODE REVIEW

File: src/ui/toCSSLineSpacing.js [hunk 1]
```diff
@@ -1,17 +1,19 @@
...
+export const LINE_SPACING_100 = '125%';
+export const LINE_SPACING_115 = '138%';
+export const LINE_SPACING_150 = '165%';
+export const LINE_SPACING_200 = '232%';
 
-// In Google Doc, "single line height" is exported as style
-// "line-height: 1.15" which is too narrow to read.
-// This defines the deprecate line spacing values that should me migrated.
-const DEPRECATED_SINGLE_LINE_SPACING_VALUES = new Set(['115%', '100%']);
+export const SINGLE_LINE_SPACING = LINE_SPACING_100;
```
Comment (velveret) [inline src/ui/toCSSLineSpacing.js]: do we need to export both `LINE_SPACING_100 / 200` and `SINGLE / DOUBLE_LINE_SPACING`?

File: src/ui/toCSSLineSpacing.js [hunk 2]
```diff
@@ -25,8 +27,26 @@ export default function toCSSLineSpacing(source: any): string {
     strValue = String(Math.round(numValue * 100)) + '%';
   }
 
-  if (DEPRECATED_SINGLE_LINE_SPACING_VALUES.has(strValue)) {
```
Comment (velveret) [inline src/ui/toCSSLineSpacing.js]: this used to handle 100% as well, do we still need a case for that?

File: src/ui/toCSSLineSpacing.js (hunk 1 above)
Comment (hedgerwang) [reply src/ui/toCSSLineSpacing.js]: Not really. I keep them for backward compatibility reason.
But I can remove them if you prefer.

File: src/ui/toCSSLineSpacing.js (hunk 2 above)
Comment (hedgerwang) [reply src/ui/toCSSLineSpacing.js]: I think so. I'll add them back.

File: src/ui/toCSSLineSpacing.js (hunk 1 above)
Comment (velveret) [reply src/ui/toCSSLineSpacing.js]: may be worth removing to reduce confusion.

**Pull request merged**